src/main/cpp/t06_homework.cpp

```diff
@@ -80,9 +80,44 @@
 
 #include "t06_homework.h"
 #include <iostream>
+#include <set>
 
 using namespace std;
 
 int t06_homework() {
+    set <string> st;
+    set <string> st_low;
+    int n = 0;
+    int counter = 0;
+    string a;
+
+    cin >> n;
+    for (int i = 0; i < n; i++) {
+        cin >> a;
+        st.insert(a);
+        for (int i = 0; i < a.length(); i++){
+            if (64 < a[i] && a[i] < 91) a[i] += 32;
+            st_low.insert(a);
+
+        }
+    }
+
+    while(cin >> a){
+        int tmp_count = 0;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The scope of the variable 'tmp_count' can be reduced.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4594561)